docs/src/main/sphinx/connector/bigquery.rst

```diff
@@ -207,10 +207,10 @@ Retrieve all records stored in the partition ``_PARTITIONDATE = '2022-04-07'``::
 SQL support
 -----------
 
-The connector provides read and write access to data and metadata in
-the BigQuery database. In addition to the :ref:`globally available
-<sql-globally-available>` and :ref:`read operation <sql-read-operations>`
-statements, the connector supports the following features:
+The connector provides read access to data and metadata in the BigQuery
+database. In addition to the :ref:`globally available <sql-globally-available>`
+and :ref:`read operation <sql-read-operations>` statements, the connector
+supports the following features:
 
 * :doc:`/sql/create-table`
 * :doc:`/sql/drop-table`
```

Comment:
I would leave the read and write as it was ... but I am also confused .. are we saying this connector can create a table and drop a table but NOT do anything else with the table .. like no INSERT or whatever .. if thats the case I question the logic of adding create table support at all .. and would add support for INSERT and others as soon as possibiel.